lib/modules/datasource/golang-version/index.ts

```diff
@@ -27,22 +28,29 @@ export class GolangVersionDatasource extends Datasource {
     'https://raw.githubusercontent.com/golang/website/',
```

Comment:
```suggestion
    'https://raw.githubusercontent.com/golang/website',
```

To align this with the returned/snapshotted value